components/template/vue/vueIndex.ts

```diff
@@ -0,0 +1,25 @@
+export default `<template>
```

Comment:
I like this a lot!

Please move to a ```/snapshots/``` folder.
For a minute there I thought the component had a second index file!

Reply:
also I'd mark this as a test file, but I tried it locally and the test gets stuck on 'running specs' 😆
@GiladShoham

Reply:
moved

Reply:
nice!